apps/prairielearn/src/components/QuestionScore.html.ts

```diff
@@ -414,3 +427,95 @@ export function ExamQuestionAvailablePoints({
     </button>
   `;
 }
+
+function QuestionValue({
+  instance_question,
+  assessment_question,
+}: {
+  instance_question: InstanceQuestion;
+  assessment_question: AssessmentQuestion;
+}) {
+  const initAutoPoints =
+    (assessment_question.init_points ?? 0) - (assessment_question.max_manual_points ?? 0);
+
+  const currentAutoValue =
+    (instance_question.current_value ?? 0) - (assessment_question.max_manual_points ?? 0);
+
+  const bestCurrentScore = run(() => {
+    const variantPoints = instance_question.variants_points_list.at(-1);
+    if (variantPoints == null) return 0;
+    if (variantPoints < initAutoPoints) return (variantPoints / initAutoPoints) * 100;
+    return 0;
+  });
+
+  const popoverContent = run(() => {
+    const parts: HtmlValue[] = [
+      'This question awards partial credit if you continue getting closer to the correct answer.',
+    ];
+
+    if (bestCurrentScore === 0) {
+      parts.push(html`<hr />`);
+      parts.push(
+        `If you score 100% on your next submission, you will be awarded an additional ${formatPoints(currentAutoValue)} points.`,
+      );
```

Comment:
Based on the recent lesson from the staff info panel, I would prefer if the `<hr>` tags are not treated as individual parts, as this may end up causing duplicated or missing breaks in future changes. I think it's safer to have the `<hr>` tags as the second argument of `joinHtml` below.

That said, from an accessibility point of view, I'm not convinced an hr is the best option in this case. I'd say it probably makes more sense to use a sequence of paragraphs or an unordered list instead.

I won't block on either case.

Reply:
Changed to paragraphs in 8512f9e.